src/client-apps/nuxeo-mobile/how-to-customize-nuxeo-mobile-workflow-tasks-layout.md

```diff
@@ -0,0 +1,333 @@
+        topics: Nuxeo Mobile
+labels:
+    - nuxeo-mobile
+    - yachour
+    - howto
+
+toc: true
+
+---
+
+Since the version 1.13.0 on iOS and 1.11.0 on Android, workflows defined in studio can be processed by users from the mobile application. 
+
+To enable the processing of tasks from mobile, there is an additional step to this configuration, to define the layout of these tasks on mobile devices : 
+
+## Prerequesites
+
+A task layout is defined Studio/Designer. For this how to, we assume that the workflow described through this tutorial [webui workflow tasks tutorial]({{page space='STUDIO' page='web-ui-workflow-tasks'}}) is created :
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/CLIENT-APPS/Mobile/mobile_workflows_WebUI_start
+    name: mobile_WF_webui_layout.png
+    1.13#screenshot#up_to_date
+--}}
+![mobile_workflows_WebUI_start](nx_asset://bf4f6004-b61e-4e48-9042-21451f163579 ?w=400)
+
+## Layout Format
+
+The format of the layout is simply a JSON object with the list of task fields to display and properties.
+
+The default layout looks like:
+
+```json
+{
+  "name": "task id",
+  "layout": [
+    {
+      "metadata": [
+        {
+          "label": "label name",
+          "field": "studio configured field",
+          "type": "cf available fields...",
+          "properties": {
+            "property1": "value",
+            "etc."
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+The list of types is detailed below with an example. 
+ 
+
+## Steps
+
+### Add a JSON resource
+In the **Resources** tab of the Designer, tab, click on the workflow and click on “create” to add an empty file with the name of the related task and the json format :
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/CLIENT-APPS/Mobile/HOW_TO_workflows_studio_create_json
+    name: HOW_TO_workflows_studio_create_json.png
+    1.13#screenshot#up_to_date
+--}}
+![HOW_TO_workflows_studio_create_json](nx_asset://ae86da20-cd15-4327-b2b6-a89b4c081f48 ?w=400)
+
+
+
+Note that the name of this file should match the task .html file name.
+
+Add the following content to this file to enable the task to display the directory selector : 
+```json
+{
+  "name": "nuxeo-task879",
+  "layout": [
+    {
+      "metadata": [
+        {
+          "label": "Type of contract",
+          "field": "type",
+          "type": "directory",
+          "properties": {
+            "multivalued": false,
+            "required": true,
+            "placeholder": "Select the contract type...",
+            "directoryName": "type",
+            "type":"select"
+            
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+Save then deploy the package.
+
+### Test your configuration on mobile
+Open Nuxeo Mobile app and set your local url. (can be found on ip tools according to your OS. ex : 172.16.32.75:8080)
```

Comment:
Note that:
- you now need to specify `http://` when the server is not responding to `https`.
- it will work only on Android as iOS will block any `http` connection except `localhost`. For iOS, there are third party tools such as https://serveo.net/ and https://ngrok.com/ to easily have a `https` server.

It may be worth to be explained there?